<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVolumeReconstructionModule</class>
 <widget class="qSlicerWidget" name="qSlicerVolumeReconstructionModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nodeSelectorLabel">
       <property name="text">
        <string>Input volume sequence:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="InputSequenceBrowserSelector">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLSequenceBrowserNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output volume node:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="OutputVolumeSelector">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLScalarVolumeNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ROI node:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="qMRMLNodeComboBox" name="ReconstructionROISelector">
         <property name="nodeTypes">
          <stringlist>
           <string>vtkMRMLAnnotationROINode</string>
          </stringlist>
         </property>
         <property name="noneEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ROIVisibilityButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../qSlicerVolumeReconstructionModule.qrc">
           <normaloff>:/Icons/VisibleOff.png</normaloff>
           <normalon>:/Icons/VisibleOn.png</normalon>:/Icons/VisibleOff.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Output image spacing for each dimension of the reconstructed volume.</string>
       </property>
       <property name="text">
        <string>Output spacing:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDoubleSpinBox" name="XSpacingSpinbox">
         <property name="toolTip">
          <string>Output image spacing for &quot;I&quot; dimension of the reconstructed volume.</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="YSpacingSpinbox">
         <property name="toolTip">
          <string>Output image spacing for &quot;J&quot; dimension of the reconstructed volume.</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ZSpacingSpinbox">
         <property name="toolTip">
          <string>Output image spacing for &quot;K&quot; dimension of the reconstructed volume.</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interpolation mode:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="InterpolationModeComboBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Optimization mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="OptimizationModeComboBox"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Compounding mode:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="CompoundingModeComboBox"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fill holes:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="FillHolesCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Number of threads:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="NumberOfThreadsSpinBox">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Skip interval:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qMRMLCollapsibleButton" name="MRMLCollapsibleButton">
     <property name="text">
      <string>Clipping</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="toolTip">
         <string> Crop rectangle origin (in pixels).</string>
        </property>
        <property name="text">
         <string>Clip rectangle origin:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSpinBox" name="XClipRectangleOriginSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string> Crop rectangle origin X component (in pixels).</string>
          </property>
          <property name="minimum">
           <number>-999999999</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="YClipRectangleOriginSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string> Crop rectangle origin Y component(in pixels).</string>
          </property>
          <property name="minimum">
           <number>-999999999</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="XClipRectangleSizeSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Crop rectangle size X component (in pixels).</string>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="YClipRectangleSizeSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Crop rectangle size Y component (in pixels).</string>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="toolTip">
         <string>Crop rectangle size (in pixels).</string>
        </property>
        <property name="text">
         <string>Clip rectangle size:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Fan angles</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="1">
      <widget class="QPushButton" name="ApplyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCollapsibleButton</class>
   <extends>ctkCollapsibleButton</extends>
   <header>qMRMLCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerVolumeReconstructionModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerVolumeReconstructionModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputSequenceBrowserSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeReconstructionModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeReconstructionModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ReconstructionROISelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
